### PR TITLE
Relax NG schema validation not working when DOCTYPE declaration missing

### PR DESCRIPTION
**Pull request by angelozerr**
Relax NG schema validation not working when DOCTYPE declaration missing with file association

Fixes https://github.com/redhat-developer/vscode-xml/issues/831

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/relaxng/xml/completion/XMLFileAssociationCompletionBasedOnRelaxNGCompactSyntaxTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/**
+ *  Copyright (c) 2018 Angelo ZERR.
```
Comment (datho7561) [inline org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/relaxng/xml/completion/XMLFileAssociationCompletionBasedOnRelaxNGCompactSyntaxTest.java:2]: 2022

**Review by datho7561 — CHANGES_REQUESTED** [review]
Works well, just noticed the date on a copyright header was wrong

File: org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/relaxng/xml/completion/XMLFileAssociationCompletionBasedOnRelaxNGCompactSyntaxTest.java (hunk 1 above)
Comment (angelozerr) [reply org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/relaxng/xml/completion/XMLFileAssociationCompletionBasedOnRelaxNGCompactSyntaxTest.java]: fixed

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks Angelo!

**Pull request merged**